app/uk/gov/hmrc/organisationsdetailsapi/connectors/IfConnector.scala

```diff
@@ -43,6 +43,10 @@ class IfConnector @Inject()(
 
   private val baseUrl = servicesConfig.baseUrl("integration-framework")
 
+  val emptyEmployeeCountResponse = EmployeeCountResponse(None, None, Some(Seq()))
+  val emptyCtReturn              = CorporationTaxReturnDetailsResponse(None, None, None, Some(Seq()))
+  val emptySaReturn              = SelfAssessmentReturnDetailResponse(None, None, None, None, Some(Seq()))
```

Comment:
Why a Some at the end and not a None? (both here and line 46)

Reply:
`None` gives no list where `some(Seq())` gives an empty list. An empty list looks better where there is no data.